plugins/colorschemes/nightfox.nix

```diff
@@ -14,6 +14,25 @@ helpers.neovim-plugin.mkNeovimPlugin config {
 
   maintainers = [ maintainers.GaetanLepage ];
 
+  colorscheme = null;
+  extraOptions = {
+    flavor = mkOption {
+      type = types.enum [
+        "carbonfox"
+        "dawnfox"
+        "dayfox"
+        "duskfox"
+        "nightfox"
+        "nordfox"
+        "terafox"
+      ];
+      example = "dayfox";
+      default = "nightfox";
+      description = "Which palette/flavor to use as the colorscheme.";
+    };
+  };
+  extraConfig = cfg: { colorscheme = mkDefault cfg.flavor; };
```

Comment:
Will this `mkDefault` reliably override the other `mkDefault`?

Reply:
The other `mkDefault` won't happen because I set `colorscheme = null` a few lines above.